lib/axiom/fs/nested_stdio.js

```diff
@@ -28,10 +28,11 @@ export var NestedStdio = function(parentStdio) {
   var ttyin = new ReadableStreamForwarder(parentStdio.ttyin);
   // Note: stdout, stderr and ttyout don't need to be wrapped as they don't
   // expose events.
+  var overlay = parentStdio.overlay;
```

Comment:
As per other comment, i'd replace this with a generic for loop that copy parent properties